### PR TITLE
Support unregistration of components

### PR DESCRIPTION
**Pull request by mcculls**
http://bamboo.s/browse/OSS-SIESTAF11-1 :white_check_mark:

### CODE REVIEW

File: siesta-server/src/main/java/org/sonatype/siesta/server/internal/resteasy/SisuResteasyDeployment.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2007-2014 Sonatype, Inc. All rights reserved.
```
Comment (kellyrob99) [inline siesta-server/src/main/java/org/sonatype/siesta/server/internal/resteasy/SisuResteasyDeployment.java]: copyright is out of date

**Comment by kellyrob99** [comment]
+1 and we could deal with the copyright along with https://issues.sonatype.org/browse/NEXUS-8356?jql=text%20~%20%22copyright%22%20and%20project%20%3D%20%22Dev%20-%20Nexus%22 to avoid the 'staleness' we commonly have in this area

File: siesta-server/src/main/java/org/sonatype/siesta/server/internal/resteasy/SisuResteasyDeployment.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.siesta.server.internal.resteasy;
+
+import org.jboss.resteasy.spi.ResteasyDeployment;
+
+/**
+ * Sisu {@link ResteasyDeployment}.
+ *
+ * @since 2.0
```
Comment (jdillon) [inline siesta-server/src/main/java/org/sonatype/siesta/server/internal/resteasy/SisuResteasyDeployment.java]: probably should be since 2.1 or 2.1.1

**Comment by jdillon** [comment]
+1 should update @since can leave headers for larger change

**Comment by mcculls** [comment]
Merged to master as 6adeace

**Pull request closed without merging**